ios/ios/ContentView.swift

```diff
@@ -0,0 +1,16 @@
+import SwiftUI
+import shared
+
+struct ContentView: View {
+	let greet = Greeting().greeting()
+
+	var body: some View {
+		Text(greet)
+	}
+}
+
+struct ContentView_Previews: PreviewProvider {
+	static var previews: some View {
+		ContentView()
+	}
+}
```

Comment:
<img width="824" alt="Screen Shot 2022-11-22 at 11 21 53" src="https://user-images.githubusercontent.com/6356137/203220727-0cc8e635-01ba-47b0-9aa3-2627230fc248.png">

It seems you are using tab as the editor settings, you can follow this settings to match iOS's chapter default. If you are not focusing on iOS yet then you can skip it.

Reply:
@blyscuit This is generated project by the IDE 🙏 I will fix them in the following PRs.